### PR TITLE
[ACM-6751] Updated uninstall topic for MCH

### PR DESCRIPTION
**Pull request by dislbenn**
Removing deprecated resources from uninstall topic.

### CODE REVIEW

**Review by jc-berger — APPROVED** [review]
/lgtm

**Comment by jc-berger** [comment]
For Jira Issue 6751: https://issues.redhat.com/browse/ACM-6751 

Specifically, modifying and updating script in step 4 of the document. See the current version here: https://github.com/stolostron/rhacm-docs/blob/2.9_stage/install/uninstall.adoc#removing-resources-by-using-commands

File: install/uninstall.adoc [hunk 1]
```diff
@@ -105,17 +105,12 @@ oc get mch -o yaml
...
+oc delete crd klusterletaddonconfigs.agent.open-cluster-management.io placementbindings.policy.open-cluster-management.io policies.policy.open-cluster-management.io userpreferences.console.open-cluster-management.io discoveredclusters.discovery.open-cluster-management.io discoveryconfigs.discovery.open-cluster-management.io
+oc delete mutatingwebhookconfiguration ocm-mutating-webhook managedclustermutators.admission.cluster.open-cluster-management.io multicluster-observability-operator
+oc delete validatingwebhookconfiguration channels.apps.open.cluster.management.webhook.validator application-webhook-validator multiclusterhub-operator-validating-webhook ocm-validating-webhook multicluster-observability-operator multiclusterengines.multicluster.openshift.io
 ----
 +
 Replace `<namespace>` in the script with the name of the namespace where {product-title-short} was installed. Ensure that you specify the correct namespace, as the namespace is cleaned out and deleted.
```
Comment (cadawson) [inline install/uninstall.adoc]: Something not related to your change here: Can we make this an *Important:* note about deleting the namespace? The concern is that if they delete the wrong namespace, they might lose data. I ran it by Mikela, and she agreed that moving it to the following line and adding the bold important callout would be beneficial. Thanks!

Comment (cadawson) [inline install/uninstall.adoc]:
```suggestion
Replace `<namespace>` in the script with the name of the namespace where {product-title-short} was installed. 

*Important:* Ensure that you specify the correct namespace, as the namespace is cleaned out and deleted. 
```

**Review by cadawson — APPROVED** [review]
Your updates look good. Added a note and an update to some other content. I don't need to review again. Thanks!

File: install/uninstall.adoc (hunk 1 above)
Comment (dislbenn) [reply install/uninstall.adoc]: @cadawson, yes I agree with that suggestion. I will make those changes.

Comment (dislbenn) [reply install/uninstall.adoc]: @cadawson @jc-berger changes has been made.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**